src/main/java/com/gmail/yuramitryahin/Main.java

```diff
@@ -1,17 +1,41 @@
 package com.gmail.yuramitryahin;
 
 import com.gmail.yuramitryahin.lib.Injector;
+import com.gmail.yuramitryahin.model.CinemaHall;
 import com.gmail.yuramitryahin.model.Movie;
+import com.gmail.yuramitryahin.model.MovieSession;
+import com.gmail.yuramitryahin.service.CinemaHallService;
 import com.gmail.yuramitryahin.service.MovieService;
+import com.gmail.yuramitryahin.service.MovieSessionService;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
 
 public class Main {
     private static Injector injector = Injector.getInstance("com.gmail.yuramitryahin");
 
     public static void main(String[] args) {
+
```

Comment:
main() method also should not contain empty lines, mentors are rejecting it

Reply:
fixed